### PR TITLE
Break greasing recommendations into sub-sections

### LINKED ISSUE #20: Make "Considerations for Applying Greasing" a bulleted list, or sub-sections?
Reading through [Considerations for Applying Greasing](https://www.ietf.org/archive/id/draft-edm-protocol-greasing-03.html#name-considerations-for-applying), I'm thinking it might be nice to have this be a bulleted list or titled subsections, so it's clear how many distinct considerations and recommendations there are?

### PR DESCRIPTION
**Pull request by LPardue**
Closes #20

### CODE REVIEW

File: draft-edm-protocol-greasing.md [hunk 1]
```diff
@@ -100,12 +100,16 @@ Implementations that receive grease values are required to ignore them. More
 background to this approach is given in {{Section 3.3 of ?VIABILITY=RFC9170}}.
 This section provides concrete suggestions for its usage.
 
+## Catch-all Handling
```
Comment (tfpauly) [inline draft-edm-protocol-greasing.md:103]:
```suggestion
## Don't Special-Case Grease Values
```

File: draft-edm-protocol-greasing.md [hunk 2]
```diff
@@ -115,11 +119,15 @@ protocol constraints. For instance, protocols that use 8-bit fields may find it
...
 likely to have no limitations.
 
+## Use Grease Values at Unpredictable Times
+
 It is recommended that senders use grease values at unpredictable times or
 sequence points during protocol interactions. This avoids receivers
 unintentionally ossifying on the occurrence of greasing in the temporal or
 spatial domain.
 
+## Defining and Registering Grease Value Ranges
```
Comment (dthaler) [inline draft-edm-protocol-greasing.md:129]:
```suggestion
## Define and Register Grease Value Ranges
```
Use part of speech consistent with the other headings added

File: draft-edm-protocol-greasing.md [hunk 3]
```diff
@@ -100,12 +100,16 @@ Implementations that receive grease values are required to ignore them. More
 background to this approach is given in {{Section 3.3 of ?VIABILITY=RFC9170}}.
 This section provides concrete suggestions for its usage.
 
+## Don't Special-Case Grease Values
+
```
Comment (gorryfair) [inline draft-edm-protocol-greasing.md:104]: Is this clearer as: Don't make Grease Values a Special Case?

File: draft-edm-protocol-greasing.md (hunk 2 above)
Comment (tfpauly) [reply draft-edm-protocol-greasing.md:129]: I was thinking the same =)

**Review by dthaler — APPROVED** [review]

File: draft-edm-protocol-greasing.md (hunk 3 above)
Comment (LPardue) [reply draft-edm-protocol-greasing.md:104]: I took the suggestion and mashed it up a little

**Pull request merged**